scene/2d/audio_stream_player_2d.cpp

```diff
@@ -186,7 +187,14 @@ void AudioStreamPlayer2D::_update_panning() {
 		float multiplier = Math::pow(1.0f - dist / max_distance, attenuation);
 		multiplier *= Math::db2linear(volume_db); //also apply player volume!
 
-		float pan = CLAMP((relative_to_listener.x + screen_size.x * 0.5) / screen_size.x, 0.0, 1.0);
+		float pan = relative_to_listener.x / screen_size.x;
+		// Don't let the panning effect extend (too far) beyond the screen.
+		pan = CLAMP(pan, -1, 1);
+
+		float global_pan_strength = GLOBAL_DEF("audio/general/2d_panning_strength", 0.5f);
```

Comment:
Querying a project setting every frame should be avoided for performance reasons, since string comparisons are involved. I'd prefer if we add a setter/getter method to AudioServer instead.

Reply:
Project settings definitely seem like the right place for this, I don't want to have AudioServer require initialization by users on game startup if they want to modify the defaults, that just feels kind of wrong.